Makefile

```diff
@@ -165,6 +165,16 @@ check-scripts:
     # Fail if any of these files have warnings
 	scripts/shellcheck.sh
 
+## check-version:           run check to ensure version in CHANGELOG.md matches references in files
```

Comment:
this description is not what is happening

Reply:
Updated to use the `-s` option explicitly.